### DISCUSSION TITLE
Word Wrap not working as expected/desired

### DISCUSSION THREAD

**Discussion by EddiesTech** [tailwindlabs/tailwindcss#10449] [Help] [answered]
Hello. I am working on my site https://gcse.work/. I am struggling to get word wrap to behave as I wish. Here are all the options I have tried and what I want/expect. What am I doing wrong and what (even if vanilla css) can I do to get my desired word wrap?

break-normal:
![image](https://user-images.githubusercontent.com/13002124/215616879-96d125cd-cc2a-4923-90c5-7905f5cf49f2.png)
break-words:
![image](https://user-images.githubusercontent.com/13002124/215616930-4c5af203-4e5f-4d50-87c5-5342eb1b1643.png)
break-all:
![image](https://user-images.githubusercontent.com/13002124/215616960-3d8fcd54-ccae-48b9-9538-67c71486487f.png)
Desired:
![image](https://user-images.githubusercontent.com/13002124/215617010-c2173af8-cd6a-42e1-8864-33ea94c60e15.png)

Many thanks in advance
Eddie

**Comment by wongjn** [comment, marked as answer]
Assuming this is in regards to the pills with the rounded corners, try applying `white-space: nowrap` to each of the `<span>` elements via the `whitespace-nowrap` utility class.

**Reply by EddiesTech** [reply]
Brilliant! With break-all on the main div and whitespace-nowrap, this works perfectly. Consider yourself added to the humans.txt: https://gcse.work/humans.txt (details: https://humanstxt.org/)
Many thanks for the prompt and useful help... you learn something every day! Now time to read on the difference between text wrapping and whitespace!
